addons/mrp/views/mrp_production_views.xml

```diff
@@ -125,11 +126,11 @@
                                     <button type="action"
                                         name="%(mrp.action_change_production_qty)d"
                                         context="{'default_mo_id': id}"
-                                        string="Update" class="oe_link" attrs="{'invisible': ['|', ('state', 'in', ('draft', 'done','cancel')), ('id', '=', False)]}"/>
+                                        string="Update" class="oe_link" attrs="{'invisible': ['|', '|', ('state', 'in', ('draft', 'done','cancel')), ('id', '=', False), ('bom_id', '=', False)]}"/>
```

Comment:
I would prefer adapt the change qty wizard :no_good:

Reply:
nope